### PR TITLE
msvc: correct usage comments

### PR DESCRIPTION
**Pull request by shuffle2**
appear to have been incorrect for a while

### CODE REVIEW

File: .private/wbs.txt [hunk 1]
```diff
@@ -15,14 +15,11 @@ o Visual Studio:
     Also make sure that the directory where libusb-1.0.lib resides is added to
     'Additional Library Directories' (Configuration Properties -> Linker
     -> General)
-  - If you use the static version of the libusb library, make sure that
-    'Runtime Library' is set to 'Multi-threaded DLL (/MD)' (Configuration
-    Properties -> C/C++ -> Code Generation).
-    NB: If your application requires /MT (Multi-threaded/libCMT), you need to
-    recompile a static libusb 1.0 library from source.
+  - By default, static libusb staticly links against the CRT and the dll build
```
Comment (tormodvolden) [inline .private/wbs.txt:18]: "staticly" ?

Comment (shuffle2) [reply .private/wbs.txt]: ha, thanks, fixed.

**Review by tormodvolden — APPROVED** [review]

**Review by mcuee — APPROVED** [review]

**Comment by mcuee** [comment]
@shuffle2 
Please help to fix the github warning.
> Merging is blocked
> The base branch requires all commits to be signed.

**Comment by shuffle2** [comment]
is that why you have been cherry-picking commits into this repo?

**Comment by mcuee** [comment]
> is that why you have been cherry-picking commits into this repo?
> 
> It doesn't look like any of the libusb commits are signed, so you should just disable the github check.

You are right. Please ignore my request. 
https://github.com/libusb/libusb/pull/1080#issuecomment-1120379433

@tormodvolden Can we disable this check? It is an eye sore...

**Comment by tormodvolden** [comment]
> @tormodvolden Can we disable this check? It is an eye sore...

I had the impression that it was enabled by @hjelmn but I am not sure.

**Pull request closed without merging**